timesketch/frontend-ng/src/store.js

```diff
@@ -239,6 +242,19 @@ export default new Vuex.Store({
         })
         .catch((e) => {})
     },
+    updateEventLabels(context, payload) {
```

Comment:
You can also get the labels from the context with `context.state.meta.filter_labels`. Then you don't need to pass them here. I saw that you are always passing the labels from the state so far.